src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars--0">
+        <div class="stars__star"></div>
```

Comment:
```suggestion
      <div class="stars stars--0">
        <div class="stars__star"></div>
```
its should work like this, BEM